crates/lox_core/src/coords/two_body.rs

```diff
@@ -236,15 +291,15 @@ mod tests {
     use super::*;
     use crate::bodies::Earth;
     use crate::frames::Icrf;
-    use crate::time::continuous::{Time, TimeScale};
+    use crate::time::continuous::{Time, TDB};
     use crate::time::dates::Date;
     use crate::time::utc::UTC;
 
     #[test]
     fn test_cartesian() {
         let date = Date::new(2023, 3, 25).expect("Date should be valid");
         let utc = UTC::new(21, 8, 0).expect("Time should be valid");
-        let time = Time::from_date_and_utc_timestamp(TimeScale::TDB, date, utc);
+        let time = Time::<TDB>::from_date_and_utc_timestamp(date, utc);
```

Comment:
This syntax makes me unhappy. I'd rather have the "useless" function argument also to keep the Rust and Python API more similar.